net/src/connection/json_protocol.rs

```diff
@@ -177,6 +179,9 @@ pub struct FetchEntryResultData {
     pub entry_address: Address,
     #[serde(rename = "content")]
     pub entry_content: serde_json::Value,
+
+    #[serde(rename = "chainHeader")]
+    pub headers: Vec<ChainHeader>,
```

Comment:
As @Connoropolous stated, this change would need a change to n3h as well (@ddd-mtl, @neonphog).

But I don't think we need to add this here. `entry_content` already includes the entry with meta (its a serialized `EntryWithMeta` struct). It could be an `EntryWithMetaAndHeader` ;)